Classes/Login/GithubClient+AccessToken.swift

```diff
@@ -27,8 +27,9 @@ extension GithubClient {
         let headers = [
             "Accept": "application/json"
         ]
+        
         request(Request(
-            url: "https://github.com/login/oauth/access_token",
+            url: Client().loginOauthUrl, // Assumes GitHub.com due to login flow 
```

Comment:
There are some places where I'm creating a new `Client()` this is all around the login flow where the `GitHubClient` doesn't have a reference to the `userSession` and therefore hasn't got a client set.

These will have to be updated once we actually have enterprise login working, but I can't make any assumptions on that work at this stage!